integration-tests/stateful-pallet-storage/handleItemized.test.ts

```diff
@@ -0,0 +1,179 @@
+            assert.notEqual(chainEvents["system.ExtrinsicSuccess"], undefined, "should have returned an ExtrinsicSuccess event");
+            assert.notEqual(chainEvents["transactionPayment.TransactionFeePaid"], undefined, "should have returned a TransactionFeePaid event");
+            assert.notEqual(pageUpdateEvent1, undefined, "should have returned a PalletStatefulStorageItemizedActionApplied event");
+            // Delete action
+            const idx_1: u16 = new u16(ExtrinsicHelper.api.registry, 1)
+            const remove_action_1 ={
+                "Remove": idx_1,
+            }
+            let remove_actions = [remove_action_1];
+            let itemized_remove_result_1 = ExtrinsicHelper.applyItemActions(providerKeys, schemaId, msa_id, remove_actions);
+            await itemized_remove_result_1.fundOperation();
+            const [pageUpdateEvent2, chainEvents2 ] = await itemized_remove_result_1.signAndSend();
+            assert.notEqual(chainEvents2["system.ExtrinsicSuccess"], undefined, "should have returned an ExtrinsicSuccess event");
+            assert.notEqual(chainEvents2["transactionPayment.TransactionFeePaid"], undefined, "should have returned a TransactionFeePaid event");
+            assert.notEqual(pageUpdateEvent2, undefined, "should have returned a event");
+        }).timeout(10000);
+
+        it("🛑 should fail to call applyItemizedAction with invalid schemaId", async function () {
+            const payload_1 = {
+                "message": "Hello World",
+            }
+            const add_action = {
+                "Add" : payload_1
+            }
+            let add_actions = [add_action];
+            let fake_schema_id = new u16(ExtrinsicHelper.api.registry, 999);      
+            let itemized_add_result_1 = ExtrinsicHelper.applyItemActions(providerKeys, fake_schema_id, msa_id, add_actions);
+            await itemized_add_result_1.fundOperation();
+            await assert.rejects(async () => {
+                await itemized_add_result_1.signAndSend();
+            }
+            , (err) => {
+                assert.notEqual(err, undefined, "should have returned an error");
+                return true;
+            });
+        }).timeout(10000);
+        
+        it("🛑 should fail to call applyItemizedAction with invalid schema location", async function () {
+            const payload_1 = {
+                "message": "Hello World",
+            }
+            const add_action = {
+                "Add" : payload_1
+            }
+            let add_actions = [add_action];    
+            let itemized_add_result_1 = ExtrinsicHelper.applyItemActions(providerKeys, schemaId_unsupported, msa_id, add_actions);
+            await itemized_add_result_1.fundOperation();
+            await assert.rejects(async () => {
+                await itemized_add_result_1.signAndSend();
+            }
+            , (err) => {
```

Comment:
You can put the actual error enum expected here instead of an error handler

Reply:
Done :)